modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenProperty.java

```diff
@@ -505,10 +505,6 @@ public boolean getRequired() {
         return required;
     }
 
-    public boolean compulsory() {
-        return getRequired() && !isNullable;
-    }
-
```

Comment:
@devhl-labs thanks for the PR.

I think that's a breaking change.

shall we keep it for the time being and remove it in 7.0.0 release later?

cc @OpenAPITools/generator-core-team

Reply:
I added that somewhat recently and nothing is using it in master. Do you want it left still?

Reply:
If it has not been included in any stable release, I'm happy to have it removed (or renamed)

Reply:
Okay great, I've renamed it.